gwlfe/model.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+
+import numpy as np
+
+
+class Model(object):
```

Comment:
Model seems like a confusing choice for this name, since GWLFE is a model itself.  Maybe `DataModel` or `InputModel` would be more specific?